cmd/check-sort-order/main.go

```diff
@@ -58,7 +58,7 @@ func main() {
 
 	if strings.HasSuffix(args.Path, ".vcf.gz") {
 		checkVCF(args.Path, gf)
-	} else if strings.HasSuffix(args.Path, ".bed.gz") {
+	} else if strings.HasSuffix(args.Path, ".bed.gz") || strings.HasSuffix(args.Path, ".bedGraph.gz") {
```

Comment:
since you're here. I would do: `strings.HasSuffix(strings.ToLower(args.Path), ".bedgraph.gz")`